ArduPlane/Parameters.h

```diff
@@ -135,6 +135,7 @@ class Parameters {
         k_param_trim_rc_at_start,
         k_param_hil_mode,
         k_param_land_disarm_delay,
+        k_param_land_abort_throttle_enable,
```

Comment:
this changes the enum values of several parameters which breaks existing configs
if you have flown this you may have a corrupt eeprom